incubator/hnc/third_party/open-policy-agent/gatekeeper/pkg/webhook/certs.go

```diff
@@ -54,23 +56,24 @@ func init() {
 	flag.BoolVar(&restartOnSecretRefresh, "cert-restart-on-secret-refresh", false, "Kills the process when secrets are refreshed so that the pod can be restarted (secrets take up to 60s to be updated by running pods)")
 }
 
-// AddRotator adds the CertRotator and ReconcileVWH to the manager.
-func AddRotator(mgr manager.Manager, cr *CertRotator, vwhName string) error {
+// AddRotator adds the CertRotator and ReconcileWH to the manager.
+func AddRotator(mgr manager.Manager, cr *CertRotator, vwhName string, crdNames []string) error {
```

Comment:
Seems it's better to add the vwh and crds to the `ReconcileWH`. I made the `VWHKey` and `CRDNames` into public fields so that here we can pass a `ReconcileWH` object with these two values set.

Reply:
Sorry I didn't really understand what this comment meant before. I don't think this is the best way to go for the reasons I explained in other comments.